app/vmselect/netstorage/netstorage.go

```diff
@@ -2259,9 +2259,11 @@ func (sn *storageNode) execOnConnWithPossibleRetry(qt *querytracer.Tracer, funcN
 	}
 	var er *errRemote
 	var ne net.Error
-	if errors.As(err, &er) || errors.As(err, &ne) && ne.Timeout() || deadline.Exceeded() {
+	var le *limitExceededErr
+	if errors.As(err, &le) || errors.As(err, &er) || errors.As(err, &ne) && ne.Timeout() || deadline.Exceeded() {
```

Comment:
I don't seem to find at which conditions this func could receive `limitExceededErr `. Looks like rpc actions could return either network errors or `errRemote` errors.

Reply:
Now there could be limitExceededErr when prcessBlock()
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/542f2a5cc4723d9c72be8fd44e185a55e14dff27/app/vmselect/netstorage/netstorage.go#L1686-L1690

This error will be eliminated after https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6464 merges to cluster branch.

Reply:
Reopen this since https://github.com/VictoriaMetrics/VictoriaMetrics/pull/6464 is refused.